tamr_toolbox/data_io/csv.py

```diff
@@ -23,6 +23,7 @@ def from_dataset(
     *,
     csv_delimiter: str = ",",
     columns: Optional[List[str]] = None,
+    column_name_dict: Optional[dict] = None,
```

Comment:
More specific type hint
```suggestion
    column_name_dict: Optional[Dict[str, str]] = None,
```

Reply:
Feel free to improve type hints in this way on existing code in the files that you're touching as well.